hudi-client/src/main/java/org/apache/hudi/HoodieWriteClient.java

```diff
@@ -509,13 +540,21 @@ private Partitioner getPartitioner(HoodieTable table, boolean isUpsert, Workload
 
   /**
    * Commit changes performed at the given commitTime marker.
+   *
+   * @param commitTime Commit Time handle
+   * @param writeStatuses RDD of WriteStatus to inspect errors and counts
+   * @return {@code true} if commit is successful. {@code false} otherwise
    */
   public boolean commit(String commitTime, JavaRDD<WriteStatus> writeStatuses) {
     return commit(commitTime, writeStatuses, Option.empty());
   }
 
   /**
    * Commit changes performed at the given commitTime marker.
+   * @param commitTime Commit Time handle
+   * @param writeStatuses RDD of WriteStatus to inspect errors and counts
```

Comment:
Again, not sure if we need to describe the data type in english for the parameters.. lets keep it high level?

Reply:
> Again, not sure if we need to describe the data type in english for the parameters.. lets keep it high level?

I wanted to keep context consistent, and fix all.